### PR TITLE
feat: allow exporting passages with inline style definitions

### PR DESCRIPTION
**Pull request by pnal**
This also needs to be backported to https://github.com/oat-sa/extension-tao-mediamanager/releases/tag/v12.13.0 and included to tao-community a November minor release

### CODE REVIEW

**Review by bartlomiejmarszal — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
LGTM

File: scripts/install/RegisterMediaResourcePreparer.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+/**
```
Comment (andreluizmachado) [inline scripts/install/RegisterMediaResourcePreparer.php:2]: suggestion:
 1 | ERROR | [x] Header blocks must be separated by a single blank line

**Review by andreluizmachado — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php [hunk 2]
```diff
@@ -168,6 +170,7 @@
             RegisterItemDataHandler::class,
             SetRolesPermissions::class,
             SetupMiddlewares::class,
+            [RegisterMediaResourcePreparer::class, ['service' => MediaResourcePreparer::class]]
```
Comment (pnal) [inline manifest.php:173]: Is it better do this by adding default config file instead of install script?

Comment (bartlomiejmarszal) [reply manifest.php:173]: No. I don't think default config will be sufficient in here. We should leave install script as it is.

**Pull request merged**